### DISCUSSION TITLE
Participant Age at Collection - CDE 14473376

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#44] [API Design Discussion]
caDSR Link: [14473376](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14473376%20and%20ver_nr=1)

Definition: "The age (in days) of the subject at the time of specimen collection."

Issues with CDE:
Federation members that collect patient age in years will have to perform a conversion to age in days. This conversion will not be exact as the reported age in years could be an integer that does not capture the additional age in days. Since we cannot provide an exact age in days, our reported data will not match the CDE definition.

Agreed Solution as of December 8, 2023:

- Federation does it's own harmonization and does not link to a CDE. Harmonization to convert age in years to age in days round((365.25 * Age_In_Years)).
- CDEs are being created with definition "approximate age in days". Federation open API will link to this CDE when it is created.

Other Proposed Solutions:

- Use a CDE that uses ISO 8601 definition. This will allow for Federation members to report age data in time formats that suit their data.

**Comment by claymcleod** [comment]
I have included this in #49 as a harmonized data element for samples where the element is not backed by a CDE. As you state, future plans include creating a new CDE conforming to an ISO 8601 date the describes the approximate age—I recommend we close this issue and open a new one when the time comes.

**Reply by GeoffLyle** [reply]
Agreed, we can deprecate this data element when a new CDE is created.

**Comment by GeoffLyle** [comment]
Note that the ongoing issues with this data element are similar to #37 and #45.

When updating this data element remember to those other data elements.
